### DISCUSSION TITLE
How 2 delete OpenTabletDriver

### DISCUSSION THREAD

**Discussion by breakphone** [OpenTabletDriver/OpenTabletDriver#1447] [Q&A] [answered]
So, after I installed the driver everything went normally. The only issue was my tablet wasn't detected, after that, I closed the program and deleted the application and restarted my PC so I can reinstall it. But after my PC was restarted, I opened my task manager and saw that OpenTabletDriver was still there, so Idk how to delete the program. I already checked on my "Uninstall a program" tab but it wasn't there. Please help >-<.
Extra: So I unplugged and plugged my tablet to see if it was detected (ctrl + D/Detect tablet didn't work) and my cursor changed to a star-like shape and went invisible, the function buttons were also different, pls help.
![hyhhjyt](https://user-images.githubusercontent.com/90963803/133951985-887b1b35-fc8c-4409-bbc4-eeb3c1b1f419.png)
Extra 2: After being frustrated and stressed because "OpenTabletDriver.UX.Wpf.exe" is still in my "Notification Area Icons" tab I installed the original Veikk drivers and it worked completely fine (after a few restarts and the removal of some files and my sanity), so is the driver here or is it deleted? If not then can anyone show me how to delete "OpenTabletDriver.UX.Wpf.exe"?
Tablet: Veikk S640

**Comment by jamesbt365** [comment, marked as answer]
open tablet driver isnt installed, it is downloaded, i fyou want to remove it if the "installer" was used then its on %localappdata%\opentabletdriver

**Reply by Mrcubix** [reply]
the only files to be deleted are:

- if you used the installer :
    - the installer
    - %localappdata%/opentabletdriver/
- if you used the portable release
    - the portable folder
    - %localappdata%/opentabletdriver/
